R/data.R

```diff
@@ -13,7 +13,19 @@
 #'     \item{y_b}{The binary response.}
 #'   }
 #'
-#' @source The data is generated by the create_dummy.R script.
+#' @source The data is generated by the `create_dummy.R` script.
 #'
 #' @keywords dataset
 "dummy_data"
+
+#' Survival Example Data
+#'
+#' This dataset contains survival data from the `survival`
+#' package's [survival::lung] dataset, modified to include factors for
+#' `sex` and `strata`, as well as a binary `status` variable which is 1
+#' for death and 0 for censored.
+#'
+#' @source The data is generated by the `create_survival.R` script.
+#'
+#' @keywords dataset
+"surv_dat"
```

Comment:
probably we can rename the `dummy_data` to be consistent across provided example toy data